src/index.html

```diff
@@ -5,8 +5,417 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a href="#home" class="header__img">
+        <img
+
+          src="images/logo.png"
+          alt="logo"
+        >
+      </a>
+
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__li">
```

Comment:
Rename all `nav__li` to `nav__item` and you could use the same naming in other projects.
We don't use non semantic tags in class names(such as p, ul, li, div) but great practice use semantic tags as a class names(header, nav, footer).